debug_endpoint_tests.py

```diff
@@ -58,7 +65,7 @@ def test_debug_token(self):
         assert access_token
 
         # Attempt to verify Token
-        token_introspection_response = self.app.get('/oauth/debug/introspect?access_token=%s' % access_token)
+        token_introspection_response = self.app.get('/oauth/debug/introspect/%s' % access_token)
```

Comment:
We should probably do `?token=:value` (https://tools.ietf.org/html/rfc7662)

Reply:
That RFC specifies a POST request and a slightly different response structure - I can quickly fix it up

Reply:
Just to finish out the thought, the original behavior 
```
GET /oauth/debug/introspect?access_token=
```

can be seen as compliant with https://tools.ietf.org/html/rfc7662 if you notice that:

1. https://tools.ietf.org/html/rfc7662 says "an authorization server offering token introspection MAY disallow the use of HTTP GET on the introspection endpoint" -- meaning that it also might _allow_ GET.

2. https://tools.ietf.org/html/draft-ietf-oauth-v2-bearer-18#section-2.3 indicates how to use an access token as a URI query parameter rather than a `Bearer` header. Generally I think this is a terrible practice, but for a debug endpoint it's convenient.